ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/TRUNK_GROUP_TESTS.yml

```diff
@@ -87,6 +87,37 @@ tenants:
         name: duplicate l2vlan210_with_trunk_groups not in use. Should not get configured anywhere
         trunk_groups: [ TG_NOT_MATCHING_ANY_SERVERS ]
 
+custom_connected_endpoints_keys:
+  - key: servers
+    type: custom_server
+  - key: monitor
+    type: monitor
+
+connected_endpoints_keys:
+  - key: Testing_1
+    type: Testing_Type_1
+
+Testing_1:
+  - name: testing
+    adapters:
+      - endpoint_ports: [ Nic1 ]
+        switch_ports: [ Ethernet111 ]
+        switches: [ trunk-group-tests-l3leaf1a ]
+        trunk_groups: [ TG_NOT_MATCHING_ANY_VLANS ]
+
+# This should not impact structured config.
+firewalls:
+  - name: FIREWALL01
+    rack: RackB
+    adapters:
+      - endpoint_ports: [ E0, E1 ]
+        switch_ports: [ Ethernet20, Ethernet20 ]
+        switches: [ DC1-LEAF2A, DC1-LEAF2B ]
```

Comment:
the devices are not part of the TRUNK_GROUP_TESTS so thats not really testing anything